bool arming
---
bool success
int8 result
